index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
please try to add the `alt` attribute to all your `<img>` tags, so the screen readers will have an idea about your images